sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -116,7 +116,7 @@ if(SYCL_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/npmiller/unified-runtime.git")
```

Comment:
Note: This will be reverted and the tag updated once the UR PR is merged, reviews should skip this part for now, the UR reviewing team will handle these.